bin/deprecateAdv.awk

```diff
@@ -1,16 +1,16 @@
 # read a Great_Library.txt
 # when a  UNIT_.*_PREREQ is found use the info to extract ObsoleteAdvance from Units.txt into variable OA
-# insert that info 2 lines after "Requires:"
+# insert that info 1 lines after "L:DATABASE_ADVANCES,ADVANCE_" (which is language indep.)
 
 match($0, /.*(UNIT_.*)_PREREQ.*/, s) {
     cmd = sprintf("awk -v var=%s '/{/ {a=$1}; /ObsoleteAdvance/ { if ( a ~ var ) print $2}' ./ctp2_data/default/gamedata/Units.txt", s[1])
     cmd | getline OA
     close(cmd)
     }
-/Requires:/ {
+/L:DATABASE_ADVANCES,ADVANCE_/ {
     found=1
     }
-(found++) == 3 && OA {
+(found++) == 2 && OA {
     printf("Obsolete:\n<L:DATABASE_ADVANCES,%s>%s<e>\n", OA, OA)
```

Comment:
This isn't language independent.

Reply:
Why not? `L:DATABASE_ADVANCES,ADVANCE_` has nothing language dependent.

Reply:
And what about the "Obsolete"? I wouldn't use that word in German, even so somebody claims it exists. And even if the spelling would be "Obsolet". I would translate it with "Veraltet mit:".

Reply:
I see what you mean. What I meant is that the insertion position is language independent in contrast to the former state where the insertion position was determined by the string `Requires:`. Taking such strings from a database would be nice, but I am not sure if I will find the time to realize that.